android/src/main/java/com/pointsdkrn/PointSdkRn.kt

```diff
@@ -51,4 +51,14 @@ class PointSdkRn(reactContext: ReactApplicationContext) :
         print("Not implemented")
         callback.invoke()
     }
-}
\ No newline at end of file
+}
+
+private fun environmentsMapping(env: String): PointEnvironment {
+    return when(env) {
+        "development" -> PointEnvironment.DEVELOPMENT
+        "staging" -> PointEnvironment.STAGING
+        "production" -> PointEnvironment.PRODUCTION
+        "preprod" -> PointEnvironment.PRE_PROD
+        else -> PointEnvironment.DEVELOPMENT //DEVELOPMENT IS THE DEFAULT SAME AS iOS
```

Comment:
No need for the comment 
```suggestion
        else -> PointEnvironment.DEVELOPMENT
```